templates/default/mongod.conf.erb

```diff
@@ -0,0 +1,18 @@
+# This file was generated by Chef Infra
+# Do NOT modify this file by hand.
+net:
+  port: <%= @port %>
+  bindIp: <%= @bind_ip %>
+  maxIncomingConnections: <%= @max_connections %>
+processManagement:
+  fork: <%= @forking %><% if @pid_file_path %>
```

Comment:
This should always be true. I think this is what's going on. I'd remove this parameter all together from the resource.

Reply:
Ok, removed!